testing/regress/ecl/sets3.ecl

```diff
@@ -17,7 +17,7 @@
 
 gavLib := service
     set of integer4 getPrimes() : eclrtl,pure,library='eclrtl',entrypoint='rtlTestGetPrimes',oldSetFormat;
-    set of integer4 getFibList(const set of integer4 inlist) : eclrtl,pure,library='eclrtl',entrypoint='rtlTestFibList',newset;
+    set of integer4 getFibList(const set of integer4 inlist) : eclrtl,pure,library='eclrtl',entrypoint='rtlTestFibList',newset,time;
```

Comment:
Agreed - it was the easiest change to implement.  I would expected it to be only added to a few functions (e.g., address cleaner) that a company would always want to monitor.  In general I would expect a new syntax, something like
`TIME(expr, 'name');`
to be used to add instances.